components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -125,9 +132,47 @@ void DeletePromoCodeFile(const base::FilePath& promo_code_file) {
   }
 }
 
+#if BUILDFLAG(IS_MAC)
+std::string ReadPromoCodeFromXattr() {
+  static const char kRefCodeAttr[] = "com.brave.refcode";
```

Comment:
nit:`constexpr`

Reply:
Fixed in [2ac2c5f](https://github.com/brave/brave-core/commit/2ac2c5f7d14ac1ec18a13631bc910e6276fc04e6)